src/card/basic/card-filter.html

```diff
@@ -1,11 +1,13 @@
-<button type="button" uib-dropdown-toggle class="btn btn-default">
-  {{currentFilter.label}}
-  <span class="caret"></span>
-</button>
-<ul uib-dropdown-menu class="dropdown-menu-right" role="menu">
-  <li ng-repeat="item in filter.filters" ng-class="{'selected': item === currentFilter}">
-    <a role="menuitem" tabindex="-1" ng-click="filterCallBackFn(item)">
-      {{item.label}}
-    </a>
-  </li>
-</ul>
+<div uib-dropdown class="card-pf-time-frame-filter">
```

Comment:
Yeah - it actually came in from master (it was a contributed change after we branched for 4.0) - I ported it in so I wouldn't lose the change.

@dlabrecq